hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -1138,6 +1138,61 @@ vlapic_calcdest(struct acrn_vm *vm, uint64_t *dmask, uint32_t dest, bool phys, b
 	}
 }
 
+/*
+ * This function populates 'dmask' with the set of vcpus that is the possible destination
+ * when lapic is passthru.
+ */
+void
+vlapic_calcdest_lapic_pt(struct acrn_vm *vm, uint64_t *dmask, uint32_t dest, bool phys)
```

Comment:
Why we cannot use vlapic_calcdest directly, as we are using vlapic_id?

Reply:
Yes, we could refine this function, and still unify it for both later, which could list as a TODO